testing/btest/Baseline.cpp/coverage.basic-no-errors/.stderr

```diff
@@ -1,2 +1,2 @@
 ### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
-Hello, Zeek!
+<...>/basic-no-errors.test (C++), line 11: Hello, Zeek!
```

Comment:
The original baseline has "line 13".  Is it possible that the C++ variation does not account for empty lines? Skimming some of the other logs, seems to be the case there, too.

Reply:
Per the above, this is reflecting that the error message tracks the beginning of compiled-to-C++ execution, rather than the direct location of the error.